ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -291,7 +311,7 @@ export class RemoteEnvironmentService extends EnvironmentService {
         // Copy files in codeDir to remote working directory
         await executor.copyDirectoryToRemote(environmentLocalTempFolder, this.remoteExperimentRootDir);
         // Execute command in remote machine, set isInteractive=true to run script in conda environment
-        executor.executeScript(executor.joinPath(environment.runnerWorkingFolder,
+        executor.executeScript(executor.joinPath(this.remoteExperimentRootDir,
```

Comment:
@SparkSnail Can you take a look this change, I found it cause a issue when I rebase this branch.